### PR TITLE
Fixes #1433

### LINKED ISSUE #1433: [BUG] Incorrect Kalos Route Order
**Info**
 
Version: 0.8.2

**Description:**

Kalos Route 22 is unlocked after clearing Kalos Route 3 and should be the 4th ordered route in Kalos. However, it is not and is instead the 23rd ordered Route in Kalos

**Expected Behavior:**

Kalos Route Order should go 1->2->3->22->4->5...etc.

**Additional Information:**

Additional Information: Fixing this issue could make some Kalos Dungeons slightly unbalanced with respect to the Closest Route difficulty.

(Reopening issue with correct tag)

### PR DESCRIPTION
**Pull request by jk13pclick**
Hopefully I did this right this time! Fixes the Kalos Route Order.

### CODE REVIEW

**Comment by Ultima1990** [comment]
I'm not exactly sure what you are trying to do here.

First off, route 22 can be accessed this early in pokémon X and Y. It's exactly the same situation as route 22 in Kanto.
Second, just moving the route down in the file and removing the 3.1 isn't going to change its unlock requirements, which you still have set as clearing route 3. The current result of this pr will be the route still unlocks after clearing route 3, but its difficulty is now higher than route 21.

Basically I don't think this route needs fixing, and this fix doesn't fix anything.

**Comment by Ultima1990** [comment]
> I'm not exactly sure what you are trying to do here.
> 
> First off, route 22 can be accessed this early in pokémon X and Y. It's exactly the same situation as route 22 in Kanto.
> Second, just moving the route down in the file and removing the 3.1 isn't going to change its unlock requirements, which you still have set as clearing route 3. The current result of this pr will be the route still unlocks after clearing route 3, but its difficulty is now higher than route 21.
> 
> Basically I don't think this route needs fixing, and this fix doesn't fix anything.

Wait, no, I got that backwards. The route moved up and the 3.1 was _added_. Don't mind me.

**Comment by RedSparr0w** [comment]
> > I'm not exactly sure what you are trying to do here.
> > First off, route 22 can be accessed this early in pokémon X and Y. It's exactly the same situation as route 22 in Kanto.
> > Second, just moving the route down in the file and removing the 3.1 isn't going to change its unlock requirements, which you still have set as clearing route 3. The current result of this pr will be the route still unlocks after clearing route 3, but its difficulty is now higher than route 21.
> > Basically I don't think this route needs fixing, and this fix doesn't fix anything.
> 
> Wait, no, I got that backwards. The route moved up and the 3.1 was _added_. Don't mind me.

Glad you spotted it, looked exactly the same to me 😅, that's the problem when it's been moved you can't really see what exactly changed.

**Review by RedSparr0w — APPROVED** [review]
Thanks for the PR, looks good.
